### DISCUSSION TITLE
Selecting more than 2 variants on the storefront

### DISCUSSION THREAD

**Discussion by Radek-Wawrzyk** [saleor/saleor#8525] [Q&A]
Hey I have a question related to `variants logic`. Namely I would like to create variant creator for more than one variant (similar effect like on the [nike storefront](https://www.nike.com/t/react-escape-run-womens-road-running-shoes-LP3Msz/DM7229-571) - There is list of sizes (labels/buttons), and colour (images))

I've noticed that when I'm creating product with eg 2 variants then each variant is "mixed/combined" - for example if I have a color (blue) and a size (EU45) variants, then I have to assign these two properties for each variant, so the API response is `{ ...productVariantObject, name: 'blue / EU45' }`. 

So my question is: **how I could create something similar like on the Nike storefront - Selecting size & colour variants.**

Saleor version - 3.0 beta

**Comment by karolkielecki** [comment]
@Radek-Wawrzyk Hi! AFAIK, there is no official feature that covers your use case. Maybe you can try using SWATCH attributes with images for each model--but that would be a lot of work...

**Comment by a-dominiak** [comment]
What about now? :D Is it supported somehow?
